libraries/AP_Vehicle/AP_Vehicle.cpp

```diff
@@ -3,7 +3,7 @@
 #include <AP_BLHeli/AP_BLHeli.h>
 #include <AP_Common/AP_FWVersion.h>
 
-#define SCHED_TASK(func, rate_hz, max_time_micros) SCHED_TASK_CLASS(AP_Vehicle, &vehicle, func, rate_hz, max_time_micros)
+#define SCHED_TASK(func, rate_hz, max_time_micros, prio) SCHED_TASK_CLASS(AP_Vehicle, &vehicle, func, rate_hz, max_time_micros, prio)
```

Comment:
I feel like it's going to get highly confusing having just arbitrary values. I think it would be better to have a standard named set by default and order by declaration task items within a particular set.

Reply:
Can you give an example of how this scheme might work?  Looking at the top of the scheduler tables there's not really a term I can come up with which relates the items while excluding other items further down in the table.